client/packages/system/src/Item/Components/PackSize/PackSizeNumberInput.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { ReactElement, useState } from 'react';
+import {
+  NumericTextInput,
+  useDebounceCallback,
+} from '@openmsupply-client/common';
+
+// This field displays a packSize number input
+export const PackSizeNumberInput = ({
+  itemId: _itemId,
+  unitName: _unitName,
+  isDisabled,
+  packSize: initialPackSize,
+  onChange,
+}: {
+  packSize: number;
+  itemId: string;
+  unitName: string | null;
+  isDisabled?: boolean;
+  onChange: (packSize: number) => void;
+}): ReactElement => {
+  const [packSize, setPackSize] = useState(initialPackSize ?? 1);
+
+  const updater = useDebounceCallback(onChange, [onChange], 250);
+  const disabled = isDisabled || false;
+
+  // This is shared between input with drop down and without drop down
+  const packSizeNumberInput = (
+    <NumericTextInput
+      //   focusOnRender={shouldFocusInput} TODO: How should this be set?
```

Comment:
I think this can be removed, looks like we were using it to focus the pack size input when you selected `custom` pack size